lib/jazzy/config.rb

```diff
@@ -226,6 +226,17 @@ def expand_path(path)
                    'comments.',
       default: false
 
+    config_attr :lint_report,
+      description: ['Sets the path to which a lint report will be written. If',
+                    'no path is provided, no lint report will be written.',
```

Comment:
If the lint report ends up growing in complexity in the future I worry that we'd regret having enabled it by default. Thoughts?